plugin-server/src/main/services/http-server.ts

```diff
@@ -29,6 +29,12 @@ export function createHttpServer(hub: Hub, serverInstance: ServerInstance, serve
                 }
             }
 
+            // Unlike the above healthcheck, this is more of a "readiness" check that verifies that the consumer
+            // connected to the group successfully (thus being assigned a member id)
+            const mainConsumerHealthy = !serverInstance.queue || serverInstance.queue.consumerReady
```

Comment:
What if we have managed to create the queue but the consumer isn't ready yet for the first - would we kill the process prematurely?

Reply:
Healthchecks/readiness checks in general work based on retries over a longer period of time. Here's the current ECS definition for example:

```json
            "healthCheck": {
                "retries": 10,
                "command": ["CMD-SHELL", "curl -f http://localhost:6738/_health || exit 1"],
                "timeout": 30,
                "interval": 20,
                "startPeriod": 60
            }
```

Reply:
Seems a bit sketch still but willing to give it a shot.

Reply:
again this is the issue with ECS not having a readiness check - it's perfectly fine and normal for a readiness check to flake. 

Things are not ready, things are not ready, and suddenly things are ready. 

It's just annoying that currently this goes under healthcheck, but this will be nicer under the new infra